component-library/component-lib/src/lib/shared/breadcrumb/breadcrumb.component.ts

```diff
@@ -23,20 +36,55 @@ export interface IBreadcrumbConfig {
   selector: 'lib-breadcrumb',
```

Comment:
We need to change the selector to the new format: ircc-cl-lib-xxxxxx

Reply:
will do when i merge with dev